pkg/agent/model/pb/storage.pb.go

```diff
@@ -0,0 +1,366 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
can we add automation here that generates this in CICD and does a git diff to ensure people are committing updated generated code? we already have an example workflow of this